components/datatypes/data/order.schema.json

```diff
@@ -108,6 +108,14 @@
           "description": "The date/time when an order cancellation is initiated by the shopper.",
           "meta:titleId": "order##xdm:cancelDate##title##86156",
           "meta:descriptionId": "order##xdm:cancelDate##description##47786"
+        },
+        "xdm:orderCustomAttributes": {
```

Comment:
@websterlove , We generally stay away from over generic data constructs as it can be used as a get away object to pass in anything without doing a schema change. We do not recommend that as we do not have strong enforcement rules at the data level.